blockstack/blockstack-app
# fix: tweaks to get extension working

Took a bit of trial and error, but overall I was able to get the extension running without too many changes in the end. I made a new webpack config file, because I had to disable code splitting and chunking. Or, at least, I couldn't figure out how to get it to work well with the extension, and it's not necessary for an extension.

To test:
- run `yarn lerna run prod:ext`
- go to chrome://extensions , turn on developer mode, then "load unpacked"
- select the the 'dist' folder inside `app`, i.e. `packages/app/dist`
- run the test app (i.e. `yarn dev`)
- log in, you should be viewing the extension now
- auth should work

I haven't done super extensive testing, and the way redux is setup might have a few more errors, as we have to write manual deserializers for the extension store.
